### PR TITLE
task-3: init

### PR DESCRIPTION
**Pull request by AverchenkoAA**
## FE
https://dwgz1q98lr4ie.cloudfront.net/

Done in MR - https://github.com/AverchenkoAA/nodejs-aws-fe/pull/2
## BE
https://mxuyp1483j.execute-api.eu-west-1.amazonaws.com/dev/pizzas

See swagger.json to more info about endpoints. You can execute it into swagger editor.

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+      - http: 
+          path: pizzas
+          method: get
+          cors: true
+  getById:
+    handler: dist/handlers/getPizzaByID.getPizzaByID
+    events:
+      - http: 
+          path: pizzas
+          method: post
```
Comment (blohamen) [inline serverless.yml:83]: Actually, it's quite unusual to use post method for the request which only return(get) something.

**Review by blohamen — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "node-aws-be",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```
Comment (wwrcth) [inline package.json:7]: Don't you want to call jest here?

File: src/handlers/getPizzaByID.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { PizzaService } from '../service/service';
+
+export async function getPizzaByID(event: any) {
+    try {
+        const { id } = event.queryStringParameters;
+        const service = new PizzaService();
+        const pizza = service.getPizzaByID(id) || 'No result';
```
Comment (wwrcth) [inline src/handlers/getPizzaByID.ts:7]: Do I understand correctly that if you can't find pizza by id you'll response with 200 and 'No results' string?

I would say it's incorrect behavior. Could you please use 404 instead?

File: src/handlers/getPizzaByID.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+            statusCode: 200,
+            headers: {
+                'Access-Control-Allow-Origin': '*',
+                'Access-Control-Allow-Credentials': true,
+            },
+            body: JSON.stringify(pizza),
+        };
+        return response;
+    } catch (error) {
+        console.log(`[getPizzaByID] Error - ${error}`);
```
Comment (wwrcth) [inline src/handlers/getPizzaByID.ts:18]: Just small advise: Winston is truly cool to use for logging ;)

File: serverless.yml (hunk 1 above)
Comment (blohamen) [reply serverless.yml:83]: According to task this API should be also with get method

Comment (wwrcth) [reply serverless.yml:83]: Agree, could you please fix it with get?

File: package.json [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    "url": "git+https://github.com/AverchenkoAA/node-aws-be.git"
+  },
+  "author": "AAA",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/AverchenkoAA/node-aws-be/issues"
+  },
+  "homepage": "https://github.com/AverchenkoAA/node-aws-be#readme",
+  "devDependencies": {
+    "@types/jest": "^26.0.22",
```
Comment (blohamen) [inline package.json:22]: According to task we should configure webpack

File: serverless.yml (hunk 1 above)
Comment (AverchenkoAA) [reply serverless.yml]: Thank you. Fixed in the last commit.

File: src/handlers/getPizzaByID.ts (hunk 3 above)
Comment (AverchenkoAA) [reply src/handlers/getPizzaByID.ts]: Thank you. Fixed in the last commit.

File: src/handlers/getPizzaByID.ts (hunk 4 above)
Comment (AverchenkoAA) [reply src/handlers/getPizzaByID.ts]: Thank you. Fixed in the last commit.

File: package.json (hunk 5 above)
Comment (AverchenkoAA) [reply package.json:23]: Thank you. Fixed in the last commit.

File: package.json (hunk 2 above)
Comment (AverchenkoAA) [reply package.json]: Thank you. Fixed in the last commit.

**Pull request merged**